modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/10_notifications-list/page-definition.json

```
@@ -0,0 +1,159 @@
+{
+	"pageElement": {
+		"pageElements": [
+			{
+				"definition": {
+					"indexed": true,
+					"layout": {
+						"borderWidth": 0,
+						"marginBottom": 0,
+						"marginLeft": 0,
+						"marginRight": 0,
+						"marginTop": 0,
+						"opacity": 100,
+						"paddingBottom": 0,
+						"paddingLeft": 0,
+						"paddingRight": 0,
+						"paddingTop": 0,
+						"widthType": "Fluid"
+					}
+				},
+				"pageElements": [
+					{
+						"definition": {
+							"fragment": {
+								"key": "layout",
+								"siteKey": "AP1"
```

Please, change all the siteKey properties in all page-definitions of the Raylife AP.

```suggestion
								"siteKey": "[$GROUP_KEY$]"
```